var/SymfonyRequirements.php

```diff
@@ -634,12 +634,6 @@ function_exists('mb_strlen'),
         );
 
         $this->addRecommendation(
-            function_exists('iconv'),
```

Comment:
I made a composer update. So yes. There is the same file in last @j0k3r PR. (and in many other PR in the past).